runtime/tests/port/j9vmemTest.c

```diff
@@ -2363,26 +2464,31 @@ j9vmem_testFindValidPageSize_impl(struct J9PortLibrary *portLibrary, char *testN
 	j9vmem_find_valid_page_size(mode, &requestedPageSize, &requestedPageFlags, &isSizeSupported);
 
 #if defined(J9VM_ENV_DATA64)
-	if (TRUE == sixteenMBPageSize) {
+	if (sixteenMBPageSize) {
 		expectedPageSize = 16 * ONE_MB;
 		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
 		expectedIsSizeSupported = TRUE;
 	} else
 #endif /* defined(J9VM_ENV_DATA64) */
-	{
+	if (0 != defaultLargePageSize) {
+		expectedPageSize = defaultLargePageSize;
+		expectedPageFlags = defaultLargePageFlags;
+		expectedIsSizeSupported = (16 * ONE_MB == defaultLargePageSize) ? TRUE : FALSE;
+	} else {
 		expectedPageSize = dataSegmentPageSize;
 		expectedPageFlags = J9PORT_VMEM_PAGE_FLAG_NOT_USED;
 		expectedIsSizeSupported = (16 * ONE_MB == dataSegmentPageSize) ? TRUE : FALSE;
 	}
 
-	verifyFindValidPageSizeOutput(portLibrary, testName,
-								expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
-								requestedPageSize, requestedPageFlags, isSizeSupported);
+	verifyFindValidPageSizeOutput(
+			portLibrary, testName,
+			expectedPageSize, expectedPageFlags, expectedIsSizeSupported,
+			requestedPageSize, requestedPageFlags, isSizeSupported);
 
 #if !defined(J9VM_ENV_DATA64)
 	outputComment(PORTLIB, "\nCase %d: -Xlp:codecache:pagesize=16M with TR_ppcCodeCacheConsolidationEnabled set\n", caseIndex++);
-	/* No port library API to set the environment variable. Use setenv() */
-	rc = setenv("TR_ppcCodeCacheConsolidationEnabled", "true", 1 /*overwrite any existing value */);
+	/* No port library API to set the environment variable. Use setenv() to overwrite any existing value. */
+	rc = setenv("TR_ppcCodeCacheConsolidationEnabled", "true", 1);
```

Comment:
I think it was better with two separate comments; the latter only related to the constant `1`.
Alternatively, we could use a macro whose name makes that point clearer:
```c
	rc = setenv("TR_ppcCodeCacheConsolidationEnabled", "true", OVERWRITE_EXISTING);
```